src/garage/torch/algos/ppo.py

```diff
@@ -59,7 +62,9 @@ def __init__(self,
                  policy_ent_coeff=0.0,
                  use_softplus_entropy=False,
                  stop_entropy_gradient=False,
-                 entropy_method='no_entropy'):
+                 entropy_method='no_entropy',
+                 minibatch_size=128,
```

Comment:
this is really big -- did you find this gets the best performance?

Reply:
Yes, compare to 32, 64, and 256 minibatch_size, I got stable and good output when using minibatch size 128. (with (32, 32) hidden size)